nuxeo-distribution/nuxeo-server-hotreload-tests/src/test/java/org/nuxeo/ftest/server/hotreload/ITDevHotReloadTest.java

```diff
@@ -249,4 +251,47 @@ public void testHotReloadFileImporters() throws IOException {
         assertEquals("Foo", docType);
     }
 
+    /**
+     * @since 11.1
+     */
+    @Test
+    public void testHotReloadSchema() throws IOException {
```

Comment:
you're right, that's the reason my first `shouldComputeMimeTypeWhenBlobsSchemaIsModified` method name. my first idea was not to mention the name `MimetypeIconUpdater` here in case we change the name of this class, but I think it will not happen before a long delay. ;)